.github/ISSUE_TEMPLATE.md

```diff
@@ -1,7 +1,6 @@
 # Bug reporting
 
-A good bug report has some very specific qualities, so please read over our short document on
-[reporting bugs][report_bugs] before you submit your bug report. 
+A good bug report has some very specific qualities, so please read over our short document on [reporting bugs][report_bugs] before submitting a bug report.
```

Comment:
aha -- first person -- I see now @fanminshi. Replace the whole line with "Please read the [bug report guidelines][report-bugs] before filing an issue."